package motor_state_list.msgs;

import "jointmotor_state.proto";

message MotorStateMap {
    repeated joint_motor_state.msgs.JointMotorState joint_motor_state = 1;
}